### PR TITLE
feat: combine Views for Login

### PR DESCRIPTION
**Pull request by Rebornix-zero**
Combine UserAddView, WordSelectView, LoginView with navigation.

### CODE REVIEW

File: src/views/LoginView/UserCard.tsx [hunk 1]
```diff
@@ -1,4 +1,9 @@
+import { useNavigation } from "@react-navigation/native";
 import { Avatar, Icon, Image } from "@rneui/themed";
+import {
+  LoginTabScreenPropsGeneric,
+  StackScreenPropsGeneric,
```
Comment (rennsax) [inline src/views/LoginView/UserCard.tsx]: Always remember to clear unnecessary imports.

Comment (Eitheages) [reply src/views/LoginView/UserCard.tsx:5]: Delete `StackScreenPropsGeneric`.

Comment (rennsax) [reply src/views/LoginView/UserCard.tsx]: Delete `StackScreenPropsGeneric`.

File: src/views/LoginView/UserCard.tsx [hunk 2]
```diff
@@ -43,17 +49,26 @@ export default function UserCard({ userName, email, selected }: UserCardProps) {
...
+          <View style={{ flex: 0 }}>
+            <Icon name="adduser" type="antdesign" size={72} />
+          </View>
+          <View style={styles.info_container}>
+            <Text style={{ fontSize: 20 }}>添加用户</Text>
+          </View>
         </View>
       </View>
-    </View>
+    </TouchableOpacity>
```
Comment (rennsax) [inline src/views/LoginView/UserCard.tsx]: Use `Pressable` API.

**Review by rennsax — APPROVED** [review]
OK.

**Pull request merged**